### PR TITLE
[OTX][Unit Test] otx.mpa.deploy

### PR DESCRIPTION
**Pull request by cih9088**
### Changes
- added unit test for `otx.mpa.deploy`
- removed unused symbolic
- adapted to torch 1.13

### Results
image

### CODE REVIEW

**Review by goodsong81 — DISMISSED** [review]
LGTM. Thanks!

**Review by sungmanc — DISMISSED** [review]

File: otx/mpa/deploy/utils/utils.py [hunk 1]
```diff
@@ -26,8 +38,27 @@ def convert_batchnorm(module):
         module_output.num_batches_tracked = module.num_batches_tracked
```
Comment (eunwoosh) [inline otx/mpa/deploy/utils/utils.py:41]:
```
if hasattr(module, "qconfig"):
    module_output.qconfig = module.qconfig
```
How do you think about adding these lines at this place? I found that this part exists in `convert_sync_batchnorm()` in torch.

Comment (cih9088) [reply otx/mpa/deploy/utils/utils.py:41]: Thanks for the suggestion. Added in https://github.com/openvinotoolkit/training_extensions/pull/1659/commits/1753494e46ff2b2936d9582d34670b06e72c5a92.

**Review by eunwoosh — APPROVED** [review]
LGTM.

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**